static/js/play.js

```diff
@@ -70,14 +79,6 @@ function handleWikipediaLink(e)
             return;
         }
 
-        // Disable the other links, otherwise we might load multiple links
```

Comment:
Yeah, this code fragment disables (prevents/overrides default behavior) all clicks after an external click. If the external click is invalid, loadPage throws an uncaught error so the new click behavior stays in effect, effectively freezing the game since the players can no longer click on even valid links.